packages/docs/src/styles/_sections.scss

```diff
@@ -54,7 +61,7 @@
   display: flex;
   flex-direction: column;
   justify-content: space-between;
-  height: $pt-grid-size * 13;
+  height: $pt-grid-size * 50;
```

Comment:
It's already the case. Setting a height here will evenly space out the children so there's spacing in between them. Without this line:
![image](https://cloud.githubusercontent.com/assets/199754/22524546/81449376-e878-11e6-9d27-0b9f899a7515.png)

So what we can do instead is remove this line and add an additional rule in the same block:
```
.pt-control-group-example {
  > .pt-control.group:not(:last-child) {
    margin-bottom: $pt-grid-size * 2;
  }
}
```